src/main/java/ca/uhn/fhir/jpa/starter/Application.java

```diff
@@ -82,10 +83,29 @@ public ServletRegistrationBean overlayRegistrationBean() {
     dispatcherServlet.setContextConfigLocation(FhirTesterConfig.class.getName());
 
     ServletRegistrationBean registrationBean = new ServletRegistrationBean();
+    registrationBean.setName("tester");
```

Comment:
Just fyi, but I found that my dispatcher below didn't work until I named this one too.